README.md

```diff
@@ -1,7 +1,28 @@
-# notation-core-go
+# Notation-core-go library
```

Comment:
Seems a different pattern from the `notation-go` readme. Maybe we can try `Notation Core Golang Library`